xcm/src/v3/traits.rs

```diff
@@ -205,8 +207,8 @@ impl From<SendError> for Error {
 
 pub type Result = result::Result<(), Error>;
 
-/// Local weight type; execution time in picoseconds.
-pub type Weight = u64;
+/// Default value for the proof size weight component. Set at 64 KB.
+pub const DEFAULT_PROOF_SIZE: u64 = 64 * 1024;
```

Comment:
This is only used in the `TryFrom<OldInstruction>`, right? In which case, better put it up there next to it as a private constant.